src/test/java/edu/uc/inventorymanager/InventoryManagerApplicationItemStatusTests.java

```diff
@@ -27,6 +27,7 @@ void fetchItemStatusById_returnsItemStatusId5() {
     }
 
     private void whenSearchItemStatusWithId5() {
+
         itemStatus = itemStatusService.fetchItemStatusById(5);
     }
```

Comment:
According to this article https://rhamedy.medium.com/a-short-summary-of-java-coding-best-practices-31283d0167d3, it’s important to have a good method structure. I have moved the method down to two different lines for the better use of spaces.

Reply:
I'm not sure I understand what you mean here. What you changed here just added a blank line in the method

Reply:
The function was all on one line before, so I thought it would be better for readability purposes to move the logic into a new line.

Reply:
I'd remove the extra line.  It's not adding value.